fastapi-challenges/websocket-chat/devashmit/main.py

```
@@ -0,0 +1,69 @@
+
+    async def disconnect(self, room, ws, username):
+        if room in self.rooms:
+            self.rooms[room] = [(w,u) for w,u in self.rooms[room] if w is not ws]
+            if not self.rooms[room]: del self.rooms[room]
+        await self.broadcast(room, f"🔴 {username} left")
+
+    async def broadcast(self, room, message, exclude=None):
+        if room not in self.rooms: return
+        for ws, _ in self.rooms[room]:
+            if ws is not exclude:
+                try: await ws.send_text(message)
```

Ruff (0.15.15)

[error] Multiple statements on one line (colon)

(E701)